inference-engine/include/ie_plugin_config.hpp

```diff
@@ -296,6 +296,11 @@ DECLARE_CONFIG_KEY(DYN_BATCH_LIMIT);
  */
 DECLARE_CONFIG_KEY(DYN_BATCH_ENABLED);
 
+/**
+ * @brief Reserved configuration for Auto-Device plugin.
+ */
+DECLARE_CONFIG_KEY(AUTO_);
```

Comment:
why adding this config key? if you want to test, please add to test file config, not here

Reply:
good catch @mashoujiang ! I personally  have no much time to review secondary PRs like this, so pls take the full responsibility.

Reply:
sure, no problem :)